### DISCUSSION TITLE
Organizationの登録申請時には、ユーザ1名のみでよろしいでしょうか。

### DISCUSSION THREAD

**Discussion by TORIFUKUKaiou** [increments/qiita-discussions#530] [ヘルプ]
いつもQiitaを便利に使わせていただいております。ありがとうーーーーッ！！！ございます、です。

[Organizationの作成方法](https://help.qiita.com/ja/articles/qiita-org-create)
を見ると、

> 申請時に、Organizationに紐づけるQiitaユーザーが2名以上（申請者本人含む）必要です

と書いてあります。

最近私は https://qiita.com/organizations/haw を申請して承認していただきました。
ありがとうーーーーッ！！！　ございます。

申請時には1名のみで申請できましたし、その後もメンバーは1名のままです。

> 申請時に、Organizationに紐づけるQiitaユーザーが2名以上（申請者本人含む）必要です

のルールがなくなったのか、 https://qiita.com/organizations/haw がルール違反？、反則技？みたいなことになってしまっているのかご確認いただけますでしょうか。（ただ墓穴を掘ってしまっている……！？）

ヘルプの更新が追いついていない（私たちの組織はセーフである）ことを祈っています :pray::pray::pray::pray::pray:

**Comment by tanaka-asami415** [comment]
@TORIFUKUKaiou 
こちらこそ、いつもありがとうございます＆ご報告ありがとうございます！

申請条件は現在も変わらず2名以上（申請者本人含む）必要なのですが
調査したところ、申請フォームの不具合でした🙇💦

おそらく今後メンバーを招待する予定かと思いますので、作成いただいたOrganizationはそのままご利用いただければ幸いです。
不具合によりお手数をおかけしてしまい、大変申し訳ございません🙏

**Comment by TORIFUKUKaiou** [comment]
承知いたしましたー。ご回答いただきましてありがとうーーーーッございます！

**Comment by TORIFUKUKaiou** [comment]
しっかりメンバー2名となりました。
https://qiita.com/organizations/haw/members

**Reply by tanaka-asami415** [reply]
@TORIFUKUKaiou 
よかったです！ご報告いただきありがとうございます！✨
